egs/hub4_spanish/s5/local/chain/run_tdnn_lstm_pitch.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+
+# adapted from babel/s5d recipe. reduced the dimensions of both the TDNN layers
+# and both dimensions of the LSTM layers
+# For now, we do not include cleanup and train on all the data
+
+# by default, with cleanup:
+# local/chain/run_tdnn_lstm_pitch.sh
+# %WER 14.82 [ 1456 / 9827, 196 ins, 244 del, 1016 sub ] exp/chain_pitch/tdnn_flstm_sp_bi_online/decode_test/wer_10_1.0
+set -e -o pipefail
+
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+nj=30
+min_seg_len=1.55
+train_set=train
+gmm=tri5  # the gmm for the target data
+langdir=data/langp/tri5_ali
+num_threads_ubm=12
+nnet3_affix=_pitch  # cleanup affix for nnet3 and chain dirs, e.g. _cleaned
+
+# The rest are configs specific to this script.  Most of the parameters
+# are just hardcoded at this level, in the commands below.
+train_stage=-10
+tree_affix=  # affix for tree directory, e.g. "a" or "b", in case we change the configuration.
+tdnn_affix=  #affix for TDNN directory, e.g. "a" or "b", in case we change the configuration.
+common_egs_dir=  # you can set this to use previously dumped egs.
+
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+local/chain/run_ivector_common_pitch.sh --stage $stage \
+                                  --nj $nj \
+                                  --min-seg-len $min_seg_len \
+                                  --train-set $train_set \
+                                  --gmm $gmm \
+                                  --num-threads-ubm $num_threads_ubm \
+                                  --nnet3-affix "$nnet3_affix"
+
+
+gmm_dir=exp/$gmm
+ali_dir=exp/${gmm}_ali_${train_set}_sp_comb
+tree_dir=exp/chain${nnet3_affix}/tree_bi${tree_affix}
+lat_dir=exp/chain${nnet3_affix}/${gmm}_${train_set}_sp_comb_lats
+dir=exp/chain${nnet3_affix}/tdnn_flstm${tdnn_affix}_sp_bi
```

Comment:
let's call it just lstm, not flstm, it's more standard.
And you can remove the _bi suffix, we always build the chain systems that way now so it's redundant.